buildflow/core/app/runtime/actors/pipeline_pattern/pull_process_push.py

```diff
@@ -47,13 +50,19 @@ def as_dict(self) -> dict:
             "process_time_millis": self.process_time_millis.average_value_rate(),
             "process_batch_time_millis": self.process_batch_time_millis.average_value_rate(),  # noqa: E501
             "pull_to_ack_time_millis": self.pull_to_ack_time_millis.average_value_rate(),  # noqa: E501
+            "cpu_percentage": self.cpu_percentage.average_value_rate(),
         }
 
 
 @ray.remote
 class PullProcessPushActor(Runtime):
     def __init__(
-        self, run_id: RunID, processor: PipelineProcessor, *, log_level: str = "INFO"
+        self,
+        run_id: RunID,
+        processor: PipelineProcessor,
+        *,
+        replica_id,
```

Comment:
nit: Can you add a ReplicaID str type alias?

Reply:
maybe in the base process_pool.py actor

Reply:
done.